### DISCUSSION TITLE
FATAL EXCEPTION: firebase-installations-executor-2

### DISCUSSION THREAD

**Discussion by mohamedsogoba** [firebase/flutterfire#10390] [Q&A]
i need help i use 
firebase_messaging: ^11.2.7
firebase_core: ^1.12.0

After doing the release of my application after some test my application close automatically

02-07 01:07:14.177  5808  5936 E AndroidRuntime: FATAL EXCEPTION: firebase-installations-executor-2
02-07 01:07:14.177  5808  5936 E AndroidRuntime: Process: com.example.test, PID: 5808
02-07 01:07:14.177  5808  5936 E AndroidRuntime: java.lang.AbstractMethodError: abstract method "android.support.v4.media.a android.support.v4.media.a.m(int)"
02-07 01:07:14.177  5808  5936 E AndroidRuntime: 	at x0.e.h(SourceFile:90)
02-07 01:07:14.177  5808  5936 E AndroidRuntime: 	at x0.e.b(SourceFile:98)
02-07 01:07:14.177  5808  5936 E AndroidRuntime: 	at com.google.firebase.installations.e.e(SourceFile:31)
02-07 01:07:14.177  5808  5936 E AndroidRuntime: 	at com.google.firebase.installations.e.b(SourceFile:66)
02-07 01:07:14.177  5808  5936 E AndroidRuntime: 	at u0.a.run(Unknown Source:18)
02-07 01:07:14.177  5808  5936 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
02-07 01:07:14.177  5808  5936 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
02-07 01:07:14.177  5808  5936 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:923)

**Comment by Altanbold** [comment]
is that solved? Same error here :'(
